src/Features/Core/Portable/EmbeddedLanguages/DateAndTime/EmbeddedCompletionContext.cs

```diff
@@ -16,12 +17,30 @@ internal partial class DateAndTimeEmbeddedCompletionProvider
     {
         private readonly struct EmbeddedCompletionContext
         {
+            private static DateTime s_exampleDateTime = DateTime.Parse("2009-06-15T13:45:30.1234567");
+            private static CultureInfo s_enUsCulture = CultureInfo.GetCultureInfo("en-US");
+            private static CultureInfo s_primaryCulture = CultureInfo.CurrentCulture;
```

Comment:
Can the CurrentCulture change, for example if I go into my Windows settings while VS is running and change it?

Reply:
(I now see this code is moving but I guess question still applies.)

Reply:
i'll just grab this explicitly.